controllers/chat/group/leaveGroupMember.js

```diff
@@ -40,6 +40,9 @@ const leaveGroupMember = async (req, res) => {
         other_text: textTargetUser,
         type: 'system',
         user_id: userId,
+        only_to_user_show: userId,
+        disable_to_user: false,
+        is_from_prepopulated: true,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of new properties `only_to_user_show`, `disable_to_user`, and `is_from_prepopulated` to the message object seems to provide more control over the message visibility and behavior. However, without knowing the exact purpose of these properties, it's hard to comment on their necessity or correctness. If they are used for controlling who can see the message or whether the message can be disabled, then these additions seem reasonable.
```diff
+        only_to_user_show: userId,
+        disable_to_user: false,
+        is_from_prepopulated: true,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->